PredixMobileUIKitTests/PredixBarChartViewTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  PredixBarChartViewTests.swift
+//  PredixMobileUIKitTests
+//
+//  Created by Fouche, George (GE Healthcare) on 12/4/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+@testable import Charts
+@testable import PredixMobileUIKit
+import XCTest
+
+class PredixBarChartViewTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testLoadChartWithAnimation() {
+        let expectation = self.expectation(description: #function)
+        let barChart = PredixBarChartView()
+
+        barChart._animator = TestAnimator(animateHandler: {
+            expectation.fulfill()
+        })
+
+        var months: [String]!
+        months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+        let unitsSold = [20.0, 4.0, 6.0, 3.0, 12.0, 16.0, 4.0, 18.0, 2.0, 4.0, 5.0, 4.0]
+        let unitsBought = [10.0, 2.0, 20.0, 4.0, 5, 8.0, 9.0, 15.0, 1.0, 3.0, 10.0, 18.0]
+
+        let bar1 = Bar(unitsSold, label: "Units Sold", color: [NSUIColor.blue])
+        let bar2 = Bar(unitsBought, label: "Units Bought", color: [NSUIColor.red])
+
+        barChart.create(xAxisValues: months, bars: [bar1, bar2], showWithDefaultAnimation: true)
+        XCTAssertTrue(barChart.data?.dataSetCount ?? 0 > 0, "No datasets were loaded in prepareForInterfaceBuilder")
```

Comment:
"No datasets were loaded in prepareForInterfaceBuilder" there is no prepareForInterfaceBuilder method. This message makes no sense.